libraries/dialogs/src/main/java/com/trendyol/uicomponents/dialogs/DialogViewState.kt

```diff
@@ -32,4 +39,24 @@ data class DialogViewState(
     fun isContentVisible(): Boolean = content.isNotEmpty()
 
     fun isContentImageVisible(): Boolean = contentImage?.takeIf { it != 0 } != null
+
+    fun getTitleBackground(context: Context) = ContextCompat.getColor(context, titleBackgroundColor)
+
+    fun getTitleTextColor(context: Context) =  ContextCompat.getColor(context, titleTextColor)
+
+    fun getTitleTextPosition(): Int {
+        return when (titleTextPosition) {
+            TextPosition.CENTER -> 4
+            TextPosition.END -> 3
+            else -> 2
+        }
+    }
+
+    fun getContentTextPosition(): Int {
+        return when (contentTextPosition) {
```

Comment:
if the default position is TextPosition.Start, then FragmentArgs might use the default value instead of null